src/plugin/places/places.js

```diff
@@ -110,12 +111,13 @@ export const SourceFields = [
   Fields.LAT_DMS,
   Fields.LON_DMS,
   Fields.MGRS,
+  Fields.ALT,
   Fields.SEMI_MAJOR,
   Fields.SEMI_MINOR,
   Fields.SEMI_MAJOR_UNITS,
   Fields.SEMI_MINOR_UNITS,
-  Fields.TIME,
-  Fields.ORIENTATION
+  Fields.ORIENTATION,
+  new ColumnDefinition(Fields.TIME, RecordField.TIME)
```

Comment:
Surprised the compiler didn't complain about this, but the type here is declared as `!Array<string>`. Seems like we just pass this later on as columns to the source, which handles both, but just to be sure we have it clear here.

Reply:
Good call. I updated the type to match `ISource#setColumns`, which is where these fields are being used in `AbstractDataSourceLayerConfig`.